### PR TITLE
Story/tp65879 browse center pane

### PR DESCRIPTION
**Pull request by bakerac4**

### CODE REVIEW

File: pep/app/pods/browse/journal/volume/route.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import Route from '@ember/routing/route';
+
+export default class BrowseJournalVolume extends Route {
+    model(params: { volume_number: string }) {
+        const journalParams = this.paramsFor('browse.journal') as { pep_code: string };
```
Comment (billdami) [inline pep/app/pods/browse/journal/volume/route.ts:5]: just cleanup, but in other projects we usually type the params as an exported interface on the route file that it belongs to e.g.
```ts
// in browse/journal/route.ts
export interface BrowseJouralParams {
    pep_code: string
}
```

File: pep/app/styles/pages/browse.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    }
+    .dotted-line {
+        position: absolute;
+        border-bottom: dotted 1px $black;
+        width: 100%;
+        height: 100%;
+    }
+
+    .page {
+        width: 35px;
```
Comment (billdami) [inline pep/app/styles/pages/browse.scss:14]: should probably use `rem` for dimensions like this when we can (i.e. this is probably about `2rem`). when `rem` is used universally for styles it makes scaling the UI to different base font sizes (which we'll eventually need for the user font size preference) much easier.

File: pep/app/pods/browse/journal/index/route.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            context: controller,
+            modelName: 'volume',
+            filterList: ['sourcecode'],
+            models: model.toArray() ?? [],
+            metadata: model.meta,
+            pagingRootKey: null,
+            filterRootKey: null
+        });
+    }
+}
```
Comment (billdami) [inline pep/app/pods/browse/journal/index/route.ts:44]: missing doc blocks in this file

File: pep/app/pods/browse/journal/index/route.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+import BrowseJournalIndexController from 'pep/pods/browse/journal/index/controller';
+import Journal from 'pep/pods/journal/model';
+import Source from 'pep/pods/source/model';
+
+export default class BrowseJournalIndex extends Route {
+    sourceCode?: string;
+    journalInformation?: Journal;
+    model() {
+        const journalParams = this.paramsFor('browse.journal') as { pep_code: string };
```
Comment (billdami) [inline pep/app/pods/browse/journal/index/route.ts:14]: continuing from the other comment, you'd then be able to reuse that params interface and import it here

File: pep/app/pods/components/modal-dialogs/help-tooltip/template.hbs [hunk 5]
```diff
@@ -3,15 +3,15 @@
     @size="sm"
     @overlayClassNames="slide-card-overlay"
     @modal-class="slide-card-modal"
-    as |Modal|
+    @closable={{false}} as |Modal|
```
Comment (billdami) [inline pep/app/pods/components/modal-dialogs/help-tooltip/template.hbs:6]: fyi this modal is only used in mobile and it has a very different layout (its a "slide up card" type UI), so we might want to keep closable=true here for its improved UX, unless they explicitly asked to remove close-on-outside-tap behavior here too.

File: pep/app/pods/read/document/controller.ts [hunk 6]
```diff
@@ -133,6 +142,26 @@ export default class ReadDocument extends Controller {
...
+     * @memberof Search
+     */
+    @action
+    async onChangeSorting(sorts: string[]) {
+        if (sorts.length) {
+            return transformSearchSortToAPI(sorts);
+        } else {
+            return [];
+        }
+    }
```
Comment (billdami) [inline pep/app/pods/read/document/controller.ts:159]: unnecessary `async`

could we move the 
```ts
if(!sorts.length) {
    return [];
}
```
logic into the `transformSearchSortToAPI()` util function, turning this into just
```ts
@action
onChangeSorting(sorts: string[]) {
    return transformSearchSortToAPI(sorts);
}
```

**Review by billdami — CHANGES_REQUESTED** [review]

File: pep/app/pods/components/modal-dialogs/help-tooltip/template.hbs (hunk 5 above)
Comment (bakerac4) [reply pep/app/pods/components/modal-dialogs/help-tooltip/template.hbs:6]: They stated do it globally, so Im going to leave this for now and if they want it changed back we can easily do that

File: pep/app/pods/read/document/controller.ts (hunk 6 above)
Comment (bakerac4) [reply pep/app/pods/read/document/controller.ts:159]: IMO I don't think we should be calling that function if we don't have any sorts. What about

```
return sorts.length ? transformSearchSortToAPI(sorts) : [];
```

Comment (billdami) [reply pep/app/pods/read/document/controller.ts:159]: if you are doing this same guard in multiple places where thats used i think its worthwhile to have the logic inside it, to just make it more robust it how it handles the args its given, its not like you are breaking its type contract, you still are passing it `string[]`.

that being said, i'm fine with your proposed change above.

**Review by billdami — APPROVED** [review]

**Pull request merged**